addons/calendar/static/tests/systray_activity_menu_tests.js

```diff
@@ -1,18 +1,24 @@
 odoo.define('calendar.systray.ActivityMenuTests', function (require) {
 "use strict";
 
-const { start } = require('mail/static/src/utils/test_utils.js');
+const { afterEach, beforeEach, start } = require('mail/static/src/utils/test_utils.js');
 var ActivityMenu = require('mail.systray.ActivityMenu');
 
 var testUtils = require('web.test_utils');
 
-
 QUnit.module('calendar', {}, function () {
-
 QUnit.module('ActivityMenu', {
-    beforeEach: function () {
-        this.data = {
+    beforeEach() {
+        beforeEach(this);
+
+        Object.assign(this.data, {
             'calendar.event': {
+                fields: { // those are all fake, this is the mock of a formatter
+                    meetings: { type: 'binary' },
+                    model: { type: 'char' },
+                    name: { type: 'char', required: true },
+                    type: { type: 'char' },
+                },
```

Comment:
I specifically avoided this one because it's crap. I actually started to do it but it lead to worse.

It's named like a python model `calendar.event` but it's the result of a formatter that is only slightly related to that model, with absolutely no field matching.